x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/location_stats.tsx

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { MetricDatum, MetricTrendShape } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+import { EuiIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import { useTheme } from '@kbn/observability-plugin/public';
+import { useAnyOfApmParams } from '../../../../../hooks/use_apm_params';
+import { useFetcher, isPending } from '../../../../../hooks/use_fetcher';
+import { CLIENT_GEO_COUNTRY_NAME } from '../../../../../../common/es_fields/apm';
+import { NOT_AVAILABLE_LABEL } from '../../../../../../common/i18n';
+import { MetricItem } from './metric_item';
+import { usePreviousPeriodLabel } from '../../../../../hooks/use_previous_period_text';
+
+const getIcon =
+  (type: string) =>
+  ({
+    width = 20,
+    height = 20,
+    color,
+  }: {
+    width: number;
+    height: number;
+    color: string;
+  }) =>
+    <EuiIcon type={type} width={width} height={height} fill={color} />;
+
+const formatDifference = (value: number) => {
+  return value > 0 ? '+' + value.toFixed(2) : value.toFixed(2);
```

Comment:
nit: usually the percentage is presented rounded. I believe the decimals don't provide any value here

`123.05%`
vs
`123%`